apple/internal/partials/support/resources_support.bzl

```diff
@@ -159,7 +160,6 @@ def _asset_catalogs(
     assets_plist = None
     infoplists = []
     if not parent_dir:
-        # TODO(kaipi): Merge this into the top level Info.plist.
```

Comment:
[question] Why remove this TODO?

Reply:
I don't know. 

This is a clean cherry-pick of the original commit [6fc1952](https://github.com/bazelbuild/rules_apple/commit/6fc19520455d0e48dd616a2ca8ddc6506d28f0f7), with only formatting and docs being changed from my side (through automation).